src/dataExtraction/wsLoad/csFlatFiles_to_ws.py

```diff
@@ -20,9 +20,6 @@
 import Bio.SeqIO
 import Bio.SeqFeature
 import urllib2
-import logging
-import warnings
-import re
 
```

Comment:
These modules are no longer needed?  (I'm not sure this script is used any more anyway so it may be irrelevant.)